### PR TITLE
Implement missing SC API and cover with tests

### PR DESCRIPTION
**Pull request by skambalin**
- [x] Add blockchain node to the local test environment
- [x] Deploy DDC Bucket Contract to the node
- [x] Fill the contract with test data
- [x] Add missing contract methods
- [x] Cover all the methods with tests

### CODE REVIEW

File: ddc-test-cluster/docker-compose.yml [hunk 1]
```diff
@@ -1,6 +1,15 @@
 version: "3"
 
 services:
```
Comment (evgeny-s) [inline ddc-test-cluster/docker-compose.yml:3]: Can we switch to the redid image, which we built recently? Ask Pavel to help

File: ddc-test-cluster/redis/main.lua [hunk 2]
```diff
@@ -46,8 +50,21 @@ local function update_node_aggregates(node_aggregation_redis_key, node_public_ke
             ['totalQueriesAcked'] = (acked and log_type == 3) and 1 or 0,
             ['totalBytesSent'] = bytes_sent,
```
Comment (evgeny-s) [inline ddc-test-cluster/redis/main.lua:51]: And in this case we should remove all the lua files

File: ddc-test-cluster/docker-compose.yml (hunk 1 above)
Comment (skambalin) [reply ddc-test-cluster/docker-compose.yml:3]: Good point. I will create another PR for this and merge it to `main` branch. Then pull the changes to this one.

Comment (skambalin) [reply ddc-test-cluster/docker-compose.yml:3]: Update the test environment to use redis image

File: ddc-test-cluster/redis/main.lua (hunk 2 above)
Comment (skambalin) [reply ddc-test-cluster/redis/main.lua]: Done

**Pull request merged**